e2e/process-services-cloud/task_filters_cloud.e2e.ts

```diff
@@ -88,15 +92,22 @@ describe('Task filters cloud', () => {
 
             tasksCloudDemoPage.completedTasksFilter().clickTaskFilter();
             expect(tasksCloudDemoPage.checkActiveFilterActive()).toBe('Completed Tasks');
+
             tasksCloudDemoPage.taskListCloudComponent().getDataTable().checkContentIsDisplayed(completedTask);
         });
 
-        xit('[C289957] Should display task filter results when task filter is selected', async () => {
+        it('[C289957] Should display task filter results when task filter is selected', async () => {
```

Comment:
This test can be deleted, also from test rail. This case is already covered by C290009.